src/main/java/org/jabref/gui/ai/components/errorstate/ErrorStateComponent.fxml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.*?>
+
+<fx:root prefHeight="200.0" prefWidth="500.0" type="BorderPane" xmlns="http://javafx.com/javafx/17.0.2-ea" xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.jabref.gui.ai.components.errorstate.ErrorStateComponent">
+   <center>
+      <VBox alignment="CENTER" spacing="10.0">
```

Comment:
Why not set and fx:id to this vbox so you can use it in the controller without having to cast?